pkgs/tools/wayland/swaycwd/default.nix

```diff
@@ -0,0 +1,45 @@
+{ pkgs, nim, stdenv, lib,
+  enableShells ? [ "bash" "zsh" "fish" "sh" "posh" ]
+}:
+stdenv.mkDerivation {
+    name = "swaycwd";
+    version = "0.0.1";
+
+    src = pkgs.fetchFromGitLab {
+      owner = "cab404";
+      repo = "swaycwd";
+      rev = "aca81695ec2102b9bca6f5bae364f69a8b9d399f";
+      hash = "sha256-MkyY3wWByQo0l0J28xKDfGtxfazVPRyZHCObl9Fszh4=";
+    };
+
+    configurePhase = ''
+        {
+        echo 'let enabledShells: seq[string] = @${builtins.toJSON enableShells}'
+        echo 'export enabledShells'
+        } | tee shells.nim
```

Comment:
```suggestion
        } | tee shells.nim
        runHook postConfigure
```

Reply:
is it really necessary to have the additionally debug output of `tee`?

Reply:
Nope, don't really remember why tee.